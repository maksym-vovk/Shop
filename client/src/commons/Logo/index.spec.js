import React from 'react';
import { Logo } from './index';

describe('Logo component', () => {
  it('Logo component render', () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });
});