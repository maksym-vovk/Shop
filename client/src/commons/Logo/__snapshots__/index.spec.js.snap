// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component Logo component render 1`] = `ShallowWrapper {}`;
